src/op_mode/interfaces.py

```diff
@@ -296,6 +305,114 @@ def _get_counter_data(ifname: typing.Optional[str],
 
     return ret
 
+def _get_kernel_data(raw, ifname = None, detail = False):
+    if ifname:
+        # Check if the interface exists
+        if not interface_exists(ifname):
+            raise vyos.opmode.Error(f"{ifname} does not exist!")
+        int_name = f' dev {ifname}'
+    else:
+        int_name = ''
+
+    kernel_interface = json.loads(cmd(f'ip -j -d -s address show{int_name}'))
```

Comment:
```suggestion
    kernel_interface = json.loads(cmd(f'ip -j -d -s address show {int_name}'))
```

A space embedded into a value and omitted in the format string `show{int_name}` is an easily avoidable footgun, in my opinion. Someone edits the `int_name` assignment and the command will become invalid. A space in the format string makes any edits in `int_name` safe.

Reply:
Thanks, I've always gone back and forth on what is correct there. Good to have clarification.